mpi-proxy-split/seq_num.cpp

```diff
@@ -18,14 +18,14 @@ extern int g_world_rank;
 extern int g_world_size;
 extern int p2p_deterministic_skip_save_request;
 volatile bool ckpt_pending;
-volatile bool freepass;
 volatile phase_t current_phase;
 unsigned int current_global_comm_id;
 reset_type_t reset_type;
 
 pthread_mutex_t seq_num_lock;
 sem_t user_thread_sem;
 sem_t ckpt_thread_sem;
+sem_t freepass_sem;
```

Comment:
Eventually, we'll want a comment above here.  If this is a semaphore, do we expect to give out two free passes, and then consume each individually?  Is that the motivation?  If so, do you have an example scenario?

Reply:
Can we move the comment to before the line:
```
sem_t freepass_sem;
```